src/DefaultHandle.jsx

```diff
@@ -27,6 +27,7 @@ function DefaultHandle({
   disabled,
   handleRef,
   theme,
+  style,
```

Comment:
r-w-s passes the "styles" prop; where is this prop coming from?

Reply:
This is a rheostat-specific prop that predates RWS. Its contents also need to be flipped for RTL (notably this style prop includes the dynamically calculated left and right positions)

Reply:
You can see it passed in [here](https://github.com/airbnb/rheostat/blob/b0eba089a4daf84f6249e0845d64a48cb8f6fe85/src/Slider.jsx#L824) and [here](https://github.com/airbnb/rheostat/blob/b0eba089a4daf84f6249e0845d64a48cb8f6fe85/src/Slider.jsx#L837)

Reply:
I’m confused; is it just that these components never used the style before? If not, why does it need to be flipped?

Reply:
If you look at the code below, it was being spread onto the DOM node explicitly as an inline style. However, passing it into the css call instead allows it to leverage the auto-flipping for RTL in the interface.